packages/blade/src/components/Breadcrumb/Breadcrumb.web.tsx

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable react-native-a11y/has-valid-accessibility-role */
+import React from 'react';
+import type { BreadcrumbProps } from './types';
+import BaseBox from '~components/Box/BaseBox';
+import { Text } from '~components/Typography';
+import { makeAccessible } from '~utils/makeAccessible';
+import { getStyledProps } from '~components/Box/styledProps';
+
+const Separator = ({
+  size,
+  color,
+}: Pick<BreadcrumbProps, 'size' | 'color'>): React.ReactElement => {
+  return (
+    <Text
+      as="span"
+      color={color === 'white' ? 'surface.text.staticWhite.muted' : 'surface.text.gray.muted'}
+      size={size}
+      weight="medium"
+    >
+      /
+    </Text>
+  );
+};
+
+const listStyle = { listStyle: 'none' };
+
+const Breadcrumb = ({
+  size = 'medium',
+  color = 'primary',
+  showLastSeparator = false,
+  accessibilityLabel = 'Breadcrumb',
+  children,
+  ...styledProps
+}: BreadcrumbProps): React.ReactElement => {
+  return (
+    <BaseBox as="nav" {...getStyledProps(styledProps)}>
+      <BaseBox
+        as="ol"
+        margin="spacing.0"
+        padding="spacing.0"
+        display="flex"
+        flexWrap="wrap"
+        gap="spacing.3"
+        alignItems="center"
+        style={listStyle}
+        {...makeAccessible({ label: accessibilityLabel })}
+      >
+        {React.Children.map(children, (child, index) => {
+          return (
+            <BaseBox
+              key={index}
+              as="li"
+              display="flex"
+              alignItems="center"
+              gap="spacing.3"
+              {...makeAccessible({
+                current: (child as React.ReactElement)?.props?.isCurrentPage ? 'page' : undefined,
+              })}
+            >
+              {React.cloneElement(child as React.ReactElement, { _size: size, _color: color })}
```

Comment:
Why not use context? this feels hacky no?

Reply:
yyyeeahh could've used Context, too lazy to write the Context wala code this was easier since there's just 1 component and 1 level deep 😆